build-system/tasks/sourcemaps.js

```diff
@@ -2,12 +2,16 @@ const argv = require('minimist')(process.argv.slice(2));
 const {
   VERSION: internalRuntimeVersion,
 } = require('../compile/internal-version');
-const path = require('path');
+const {posix: path} = require('path');
 const Remapping = require('@ampproject/remapping');
+const ResolveUri = require('@jridgewell/resolve-uri');
 
 /** @type {Remapping.default} */
 const remapping = /** @type {*} */ (Remapping);
 
+/** @type {ResolveUri.default} */
```

Comment:
why aren't these two types being picked up automatically by ts?

Reply:
Because I use rollup to generate CJS friendly exports when you use `const foo = require()`, and TS's types expect` const foo = require().default`